x/crosschain/keeper/msg_server_vote_outbound_tx_test.go

```diff
@@ -203,6 +203,8 @@ func TestKeeper_VoteOutbound(t *testing.T) {
 		// Successfully mock ProcessOutbound
 		keepertest.MockGetRevertGasLimitForERC20(fungibleMock, asset, *senderChain, 100)
 		keepertest.MockPayGasAndUpdateCCTX(fungibleMock, observerMock, ctx, *k, *senderChain, asset)
+		observerMock.On("GetSupportedChainFromChainID", mock.Anything, senderChain.ChainId).
```

Comment:
Maybe we can move this to a function as well, It just one line , but might be easier to refactor if the interface changes in the future